tools/ci_build/github/download_test_data.py

```diff
@@ -17,13 +17,15 @@ def get_azure_region():
 def parse_arguments():
     parser = argparse.ArgumentParser(description="ONNXRuntime Data Downloader.")
     parser.add_argument("--test_data_url", help="Test data URL.")
+    parser.add_argument("--azure_region", help="Azure region")
     return parser.parse_args()
 
 
-def get_server_hostname():
-    #should be northcentralus or centralus
-    azure_location=get_azure_region()
-    print(azure_location)
+def get_server_hostname(azure_location):
+    if azure_location is None:
+      #should be northcentralus or centralus
+      azure_location=get_azure_region()
```

Comment:
It looks like this is mostly for optimizing download speed, depending on region. Setting up a CDN will alleviate this, and get faster downloads globally, not just US regions (https://docs.microsoft.com/en-us/azure/cdn/)